omegaconf/listconfig.py

```diff
@@ -372,7 +372,8 @@ def _get_node(
         self,
         key: Union[int, slice],
         validate_access: bool = True,
-        throw_on_missing: bool = False,
+        throw_on_missing_value: bool = False,
+        throw_on_missing_key: bool = False,
```

Comment:
Currently `ListConfig._get_node` gives the same behavior whether `throw_on_missing_key` is `True` or `False` (because the method does not make any use of this keyword argument).

Reply:
Yes, this is coming from the interface. there was no need yet to support it.
This is unrelated to this PR though, feel free to post a followup issue fixing it (there aren't a lot of uses of throw_on_missing yet, we may eventually just revert it is there isn't any need).

Reply:
_get_node is an internal API. As long as there is no external API that can expose an issue because of it I don't think it's particularly important.